### PR TITLE
fixed modify document functions in WOQL

### LINKED ISSUE #104: Not able to insert documents using WOQL.insert_document
**Describe the bug**
We get error when trying to insert documents using WOQL.insert_document

### PR DESCRIPTION
**Pull request by Neelterminusdb**
fixes #104

### CODE REVIEW

**Comment by spl** [comment]
https://github.com/terminusdb/terminusdb/pull/1013 has been merged, so I believe this can be restarted.

**Review by spl — CHANGES_REQUESTED** [review]
There are merge conflicts with `main`, and I see things like inconsistent `;` usage, so I'm wondering if this passes the lint check. I think you should merge in `main` and check.

File: lib/woqlClient.js [hunk 1]
```diff
@@ -753,6 +753,8 @@ WOQLClient.prototype.dispatch =  function(action, apiUrl, payload) {
             ),
         )
     }
+    console.log(apiUrl)
```
Comment (rrooij) [inline lib/woqlClient.js:756]: We should probably remove this

File: lib/woql.js [hunk 2]
```diff
@@ -118,6 +123,10 @@ WOQL.read_document = function (IRI, output, formatObj) {
  * have a class specified key.
  * @param {string} [IRI] - An optional identifier specifying the document location.
  * @return {object} WOQLQuery
+ * @example
+ * const res = await client.query(
+ * WOQL.insert_document(WOQL.doc({ "@type" : "Person", "label": "John" }))
```
Comment (spl) [inline lib/woql.js:128]: Indent

File: lib/woql.js [hunk 3]
```diff
@@ -1264,6 +1273,18 @@ WOQL.vars = function (...varNames) {
   return varNames.map((item) => new Var(item));
 };
 
+/**
+ * Creates a Document which can be used in WOQL within a query
```
Comment (spl) [inline lib/woql.js:1277]:
```suggestion
 * Produces an encoded form of a document that can be used by a WOQL operation such as `WOQL.insert_document`.
```

File: lib/woql.js [hunk 4]
```diff
@@ -1264,6 +1273,18 @@ WOQL.vars = function (...varNames) {
...
+ * @param  {object} object The document which needs to be created
```
Comment (spl) [inline lib/woql.js:1278]:
```suggestion
 * @param  {object} object - Document to encode
```

File: lib/woql.js [hunk 5]
```diff
@@ -1264,6 +1273,18 @@ WOQL.vars = function (...varNames) {
...
+ * @returns {object} a object with encoded document
```
Comment (spl) [inline lib/woql.js:1279]:
```suggestion
 * @returns {object} The encoded document
```

**Review by spl — CHANGES_REQUESTED** [review]

**Review by spl — APPROVED** [review]

**Pull request merged**